<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicApplication</class>
 <widget class="QWidget" name="MusicApplication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>天天酷音</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../qrc/MusicPlayer.qrc">
    <normaloff>:/image/windowicon</normaloff>:/image/windowicon</iconset>
  </property>
  <widget class="QToolButton" name="windowClose">
   <property name="geometry">
    <rect>
     <x>917</x>
     <y>7</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="minimization">
   <property name="geometry">
    <rect>
     <x>889</x>
     <y>7</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>620</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="musicPrivious">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>70</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicNext">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MusicTimerSliderWidget" name="musicTimeWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>355</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QToolButton" name="menuSetting">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>580</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="musicKey">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>70</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicSound">
   <property name="geometry">
    <rect>
     <x>208</x>
     <y>128</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/soundmax);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="musicSoundSlider">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="volumeValue">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>130</y>
     <width>30</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#FFFFFF;</string>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="songsContainer">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>330</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="MusicMarquee" name="showCurrentSong" native="true">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>550</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#FFFFFF;font-weight:bold;font-size:15px;</string>
   </property>
   <property name="text" stdset="0">
    <string>当前无正在播放的歌曲</string>
   </property>
  </widget>
  <widget class="QToolButton" name="musicSetting">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>580</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicImport">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>580</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicSearch">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>580</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicCurrentLocation">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>580</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="musicWindowTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:16px;color:#FFFFFF;font-weight:bold</string>
   </property>
   <property name="text">
    <string>天天酷音</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="MusicLocalSongSearchEdit" name="musicSongSearchLine" native="true">
   <property name="geometry">
    <rect>
     <x>484</x>
     <y>7</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="musicWindowIcon">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>5</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/windowicon);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="musicWindowSpace">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/screentop);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="musicSearchButton">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>7</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/searchlineleft);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicWindowChangeSkin">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>7</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="SurfaceStackedWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>521</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QLabel" name="showPicture">
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/beauty);</string>
    </property>
   </widget>
   <widget class="MusicSongSearchOnlineWidget" name="songSearchWidget"/>
   <widget class="MusicLrcContainerForInline" name="musiclrccontainerforinline"/>
  </widget>
  <widget class="QPushButton" name="musicSearchWidgetButton">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>60</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>音乐</string>
   </property>
  </widget>
  <widget class="QPushButton" name="musicLrcWidgetButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>歌词</string>
   </property>
  </widget>
  <widget class="QPushButton" name="musicSearchRefreshButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>55</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="musicIndexWidgetButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>首页</string>
   </property>
  </widget>
  <widget class="QLabel" name="showDownloadGif">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>480</y>
     <width>41</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="musicDesktopLrc">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>130</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#FFFFFF;</string>
   </property>
   <property name="text">
    <string>桌面歌词</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="musicBestLove">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="userWindow">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QToolButton" name="musicButton_tools">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="music3DPlayButton">
   <property name="geometry">
    <rect>
     <x>318</x>
     <y>100</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicWindowRemote">
   <property name="geometry">
    <rect>
     <x>858</x>
     <y>7</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicButton_playlist">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicButton_radio">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicButton_mydownl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicButton_cloud">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="vedioWidgetButton">
   <property name="geometry">
    <rect>
     <x>755</x>
     <y>60</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>视频</string>
   </property>
  </widget>
  <widget class="QLabel" name="playCurrentTime">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>75</y>
     <width>30</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#FFFFFF;</string>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="playTotalTime">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>75</y>
     <width>30</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#FFFFFF;</string>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QToolButton" name="musicPlayMode">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>127</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>10</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="musicWindowConcise">
   <property name="geometry">
    <rect>
     <x>828</x>
     <y>7</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <action name="action_About">
   <property name="text">
    <string>关于天天酷音(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>关于天天酷音</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="../../qrc/MusicPlayer.qrc">
     <normaloff>:/image/close</normaloff>:/image/close</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_ImportSongs">
   <property name="icon">
    <iconset resource="../../qrc/MusicPlayer.qrc">
     <normaloff>:/image/import</normaloff>:/image/import</iconset>
   </property>
   <property name="text">
    <string>导入本地歌曲(&amp;I)</string>
   </property>
   <property name="toolTip">
    <string>添加歌曲</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_Privious">
   <property name="icon">
    <iconset resource="../../qrc/MusicPlayer.qrc">
     <normaloff>:/image/privious</normaloff>:/image/privious</iconset>
   </property>
   <property name="text">
    <string>上一曲(&amp;P)</string>
   </property>
   <property name="toolTip">
    <string>上一曲</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="action_Next">
   <property name="icon">
    <iconset resource="../../qrc/MusicPlayer.qrc">
     <normaloff>:/image/next</normaloff>:/image/next</iconset>
   </property>
   <property name="text">
    <string>下一曲(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>下一曲</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="action_Play">
   <property name="icon">
    <iconset resource="../../qrc/MusicPlayer.qrc">
     <normaloff>:/image/play</normaloff>:/image/play</iconset>
   </property>
   <property name="text">
    <string>播放/暂停</string>
   </property>
   <property name="toolTip">
    <string>播放/暂停</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_VolumePlus">
   <property name="icon">
    <iconset resource="../../qrc/MusicPlayer.qrc">
     <normaloff>:/image/soundmax</normaloff>:/image/soundmax</iconset>
   </property>
   <property name="text">
    <string>音量(&amp;+)</string>
   </property>
   <property name="toolTip">
    <string>增加音量</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="action_VolumeSub">
   <property name="icon">
    <iconset resource="../../qrc/MusicPlayer.qrc">
     <normaloff>:/image/soundmin</normaloff>:/image/soundmin</iconset>
   </property>
   <property name="text">
    <string>音量(&amp;-)</string>
   </property>
   <property name="toolTip">
    <string>降低音量</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="action_OrderPlay">
   <property name="icon">
    <iconset resource="../../qrc/MusicPlayer.qrc">
     <normaloff>:/image/orderplay</normaloff>:/image/orderplay</iconset>
   </property>
   <property name="text">
    <string>顺序播放</string>
   </property>
   <property name="toolTip">
    <string>顺序播放</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
  </action>
  <action name="action_RandomPlay">
   <property name="icon">
    <iconset resource="../../qrc/MusicPlayer.qrc">
     <normaloff>:/image/randomplay</normaloff>:/image/randomplay</iconset>
   </property>
   <property name="text">
    <string>随机播放</string>
   </property>
   <property name="toolTip">
    <string>随机播放</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
  </action>
  <action name="action_SingleCycle">
   <property name="icon">
    <iconset resource="../../qrc/MusicPlayer.qrc">
     <normaloff>:/image/oneloop</normaloff>:/image/oneloop</iconset>
   </property>
   <property name="text">
    <string>单曲循环</string>
   </property>
   <property name="toolTip">
    <string>单曲循环</string>
   </property>
   <property name="shortcut">
    <string>Alt+3</string>
   </property>
  </action>
  <action name="action_ListCycle">
   <property name="icon">
    <iconset resource="../../qrc/MusicPlayer.qrc">
     <normaloff>:/image/listloop</normaloff>:/image/listloop</iconset>
   </property>
   <property name="text">
    <string>列表循环</string>
   </property>
   <property name="toolTip">
    <string>列表循环</string>
   </property>
   <property name="shortcut">
    <string>Alt+4</string>
   </property>
  </action>
  <action name="action_Setting">
   <property name="icon">
    <iconset resource="../../qrc/MusicPlayer.qrc">
     <normaloff>:/image/setting</normaloff>:/image/setting</iconset>
   </property>
   <property name="text">
    <string>设置(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>设置</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <zorder>background</zorder>
  <zorder>windowClose</zorder>
  <zorder>minimization</zorder>
  <zorder>musicPrivious</zorder>
  <zorder>musicNext</zorder>
  <zorder>menuSetting</zorder>
  <zorder>musicKey</zorder>
  <zorder>musicSound</zorder>
  <zorder>musicSoundSlider</zorder>
  <zorder>volumeValue</zorder>
  <zorder>songsContainer</zorder>
  <zorder>showCurrentSong</zorder>
  <zorder>musicSetting</zorder>
  <zorder>musicImport</zorder>
  <zorder>musicSearch</zorder>
  <zorder>musicCurrentLocation</zorder>
  <zorder>musicWindowTitle</zorder>
  <zorder>musicWindowIcon</zorder>
  <zorder>musicWindowSpace</zorder>
  <zorder>musicSongSearchLine</zorder>
  <zorder>musicSearchButton</zorder>
  <zorder>musicWindowChangeSkin</zorder>
  <zorder>SurfaceStackedWidget</zorder>
  <zorder>musicSearchWidgetButton</zorder>
  <zorder>musicLrcWidgetButton</zorder>
  <zorder>musicSearchRefreshButton</zorder>
  <zorder>musicIndexWidgetButton</zorder>
  <zorder>showDownloadGif</zorder>
  <zorder>musicDesktopLrc</zorder>
  <zorder>musicBestLove</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>musicButton_tools</zorder>
  <zorder>music3DPlayButton</zorder>
  <zorder>musicWindowRemote</zorder>
  <zorder>musicButton_playlist</zorder>
  <zorder>musicButton_radio</zorder>
  <zorder>musicButton_mydownl</zorder>
  <zorder>musicButton_cloud</zorder>
  <zorder>label</zorder>
  <zorder>musicTimeWidget</zorder>
  <zorder>vedioWidgetButton</zorder>
  <zorder>playCurrentTime</zorder>
  <zorder>playTotalTime</zorder>
  <zorder>musicPlayMode</zorder>
  <zorder>line</zorder>
  <zorder>musicWindowConcise</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MusicMarquee</class>
   <extends>QWidget</extends>
   <header>musicmarquee.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicLrcContainerForInline</class>
   <extends>QWidget</extends>
   <header>lrcmanager/musiclrccontainerforinline.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicSongSearchOnlineWidget</class>
   <extends>QWidget</extends>
   <header>musicsongsearchonlinewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicTimerSliderWidget</class>
   <extends>QWidget</extends>
   <header>musictimersliderwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicLocalSongSearchEdit</class>
   <extends>QWidget</extends>
   <header>musiclocalsongsearchedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../qrc/MusicPlayer.qrc"/>
 </resources>
 <connections/>
</ui>
